### DISCUSSION TITLE
OpenMPTCProuter bonding vs Speedify and Internet traffic usage in VPS

### DISCUSSION THREAD

**Discussion by gaamaaresosa** [Ysurac/openmptcprouter#2755] [Q&A] [answered]
1) How do you compare OpenMPTCProuter bonding with Speedify ?
     With OpenMPTCProuter Bonding be much closer to Speedify ?
     What are the features missing on each other ?

2) When I do a live video streaming to Youtube with 5Mbps form OpenMPTCProuter bonding,
     will the could server charge me for the internet traffic I use for continuous upload ?
     Eg: DigitalOcean limit per month 1TB only and I stream 5Mbps 24/7

3) I would like to do live video streaming to youtube from Raspberry Pi 4 itself with some encoder code in python.
     On the same Raspberry Pi 4 I need internet bonding also with OpenMPTCProuter.
     How can we proceed ?
     Raspbian Pi OS lite 32/64bit with video encoder and Python/Flask web interface.
     Can I install OpenMPTCProuter on VM inside Raspberry Pi 4 and access bonded internet from few 4G sim cards ?

**Comment by Ysurac** [comment, marked as answer]
1. I never tested speedify, so I can't answer here
2. Yes it will charge if you transfer more than 1TB
3. OpenMPTCProuter is based on OpenWRT, you can install all OpenWRT packages. RPI4 is too slow to have OpenMPTCProuter inside a VM.

**Reply by gaamaaresosa** [reply]
Thanks Ysurac.
Three more questions please.

1) Is it manditory to set static IP for all the 4G modems where we use the SIM cards and plug that on OpenMPTCProuter ?
Otherwise bond and aggregation won't happen ?
Keeping DHCP on 4G modems would be easiy right ?

2) Can we run OpenMPTCProuter inside Windows 10 docker ?

3) When we plugout one of the 4G modem (Huawei) while the streaming is running.
     Why there is a break on the stream and internet for a while ?
     Can't it be seamless ?

**Reply by Ysurac** [reply]
1. No, DHCP is supported, even if it's better to have static IP because DHCP renew can sometimes cut connections.
2. No, Linux kernel must support MPTCP
3. Shouldn't have any break, maybe better in next release

**Reply by gaamaaresosa** [reply]
Thanks again Ysurac Genius !
I was trying in DHCP modems, so that could be the reason for the 3)* break.
